lib/jeckle/http.rb

```diff
@@ -39,6 +41,23 @@ def resource_name
         @resource_name ||= model_name.element.pluralize
       end
 
+      # @public
```

Comment:
@brennovich Do you have any intention of documenting private methods too? If not, I don't know if I would add this.
I know it's not a big deal, it's just me being annoying.